<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vehicule</class>
 <widget class="QDialog" name="vehicule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QClient{
	background-color: #071e26;
}
QWidget {	
	border:1px solid #dfdfdf;
	border-radius: 5px;
	background-color: #ffffff;
}

QPushButton{
	
	border-raduis:5px
}
QPushButton:pressed {
	background-color: rgb(189, 189, 189);
}
#side_menu{
	border-radius: 20px;
	background-color:rgb(0, 0, 0);
}
#top_widget{

	background-color:rgb(0, 0, 0);
}
#form{
	
	
	background-color: rgb(255, 255, 255);
border-top-left-radius: 50px;
}

#widget_2 QLineEdit{
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(0,0,0);
padding-bottom:7px

}
#widget_2{
	
	/* background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);*/
	/*background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0.1 #646464, stop: 1 #5d5d5d);*/
     background-color:QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4aafdb, stop: 1 #090763);
}
#options{
background-color:rgb(0, 0, 0);
border-bottom-right-radius: 50px;

}
#widget_2 #recherche{
	
	background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);
}
#widget_2 #ajouter{
 color:rgb(0, 0, 0);
    background-color: rgb(184, 184, 184);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}


#widget_2 #modifier{
 color:rgb(0, 0, 0);
    background-color: rgb(184, 184, 184);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

#widget_2 #supprimer{
 color:rgb(0, 0, 0);
    background-color: rgb(184, 184, 184);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

#widget_2 #ajouter:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#widget_2 #modifier:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#widget_2 #supprimer:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}
#pdf_3{
	
	background-color: #071e26;
border:0px ;
}
#side_menu #stat_3{
	
	background-color: #071e26;
border:0px ;
}
#widget_2 #widget{
border-radius: 20px;
	background-color:rgb(0, 0, 0);
}
#widget QLineEdit {
background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);
}

</string>
  </property>
  <widget class="QWidget" name="top_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1861</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/vehi.png</normaloff>:/new/prefix1/icons/vehi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 160px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/retour2.png</normaloff>:/new/prefix1/icons/retour2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>1651</width>
     <height>821</height>
    </rect>
   </property>
   <widget class="QLabel" name="options">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>181</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>941</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-raduis:5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>60</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/rech.png</normaloff>:/new/prefix1/rech.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QLabel" name="form">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>id vehicule </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Ã©tat</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>841</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="recherche">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>  recherche</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>801</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>50</y>
       <width>73</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Matricule</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-5</y>
      <width>1661</width>
      <height>831</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="style.qrc">:/new/prefix1/icons/blue2.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>options</zorder>
   <zorder>tableView</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>ajouter</zorder>
   <zorder>supprimer</zorder>
   <zorder>modifier</zorder>
   <zorder>form</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>comboBox</zorder>
   <zorder>widget</zorder>
   <zorder>lineEdit_7</zorder>
  </widget>
  <widget class="QWidget" name="side_menu" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>191</width>
     <height>821</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rendez-Vous</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/rend.png</normaloff>:/new/prefix1/icons/rend.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipements</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/equi.png</normaloff>:/new/prefix1/icons/equi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>         Clients</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/cli.png</normaloff>:/new/prefix1/icons/cli.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
